fdbserver/DDTenantCache.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * DDTenantCache.h
```

Comment:
I haven't looked carefully yet, but in a quick look this didn't seem to be related to the data distributor other than that it is used by it. If that's the case, could we give it a more generic name? Possibly it would be useful to some of the other tenant users.

Reply:
Right. AS of this change, there is nothing specific to DD in the tenant cache code. I could remove the DD prefix in the name. I have an upcoming change where this cache is able to send promises on a stream when it refreshes a cache and detects a new tenant. Currently DDTracker listens on that stream and acts to split shards at tenant boundaries if there is a new tenant created. I suppose that is also generalizable to other use cases.